src/chbv.jl

```diff
@@ -52,24 +68,30 @@ Roger B. Sidje, see below.
     EXPOKIT: Software Package for Computing Matrix Exponentials.
     ACM - Transactions On Mathematical Software, 24(1):130-156, 1998
 """
-function chbv(A, vec)
+function chbv{T}(A, vec::Vector{T})
+    result = convert(Vector{promote_type(eltype(A), T)}, copy(vec))
+    chbv!(A, result)
```

Comment:
I think this will result in an unnecessary  copy. It's better to use the three argument version here.